src/kern/npf.h

```diff
@@ -310,6 +310,7 @@ typedef struct npf_ioctl_table {
 #define	IOC_NPF_SAVE		_IOR('N', 105, nvlist_ref_t)
 #define	IOC_NPF_RULE		_IOWR('N', 107, nvlist_ref_t)
 #define	IOC_NPF_CONN_LOOKUP	_IOWR('N', 108, nvlist_ref_t)
+#define	IOC_NPF_TABLE_REPLACE	_IOW('N', 109, nvlist_ref_t)
```

Comment:
One more thing: IIRC, this should be `_IOWR`, since you also read the result table.

Reply:
Done. I'm new to the ioctl backend stuff so thanks for picking this one up! :)